### ISSUE TITLE
How can I setup a schedule for exact minute of every hour?

### ISSUE THREAD

**Issue by mickey9801** [dbader/schedule#144]
I have setup a raspberry pi for read temperature and humidity from sensor and I would like it to publish those data to a MQTT broker at 00,15,30,45 minute of every hour. How can I setup this schedule?

I have setup a schedule like this: `schedule.every(15).minutes.do(read_n_publish)` but it can only run the job every 15 minute from the time I start run the program.

**Comment by cparrish60** [comment]
I have a similar requirement.

**Comment by onesuper** [comment]
+1

**Comment by fauzanM** [comment]
@mickey9801  This is really simple. 

The whole schedule package implementation is sequential. And, you could solve the problem like this. 
`schedule.every(15).minutes.do(read_n_publish)
schedule.every(30).minutes.do(read_n_publish)
schedule.every(45).minutes.do(read_n_publish)

while True:
schedule.run_pending()
time.sleep(1)`

What this will do is it will run the code every 15, 30, 45 minutes.

**Comment by dbader** [comment]
Some work on that happening here: https://github.com/dbader/schedule/pull/116

**Comment by Nilpo** [comment]
@fauzanM You would only need to schedule `every(15)`. Yours would result in stacking. It would also be contingent upon the script being started at exactly minute 00.

**Comment by rokcarl** [comment]
Exactly, so the @fauzanbaigm solution wouldn't work correctly, i.e. it wouldn't run at exactly e.g. 12:30, but it would run at e.g. 12:38.

I need this functionality as well.
